x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/services/elasticsearch/ElasticsearchInternalServiceTests.java

```diff
@@ -1275,7 +1275,8 @@ public void testChunkingLargeDocument() throws InterruptedException {
         assumeTrue("Only if 'inference_chunking_settings' feature flag is enabled", ChunkingSettingsFeatureFlag.isEnabled());
 
         int wordsPerChunk = 10;
-        int numBatches = randomIntBetween(3, 6);
+        int numBatches = 3;
+        randomIntBetween(3, 6);
```

Comment:
?

Reply:
I think this line still needs to be cleaned up?